tests/Unit/Test_ActionTesting.cpp

```diff
@@ -0,0 +1,150 @@
+                    const Parallel::ConstGlobalCache<Metavariables>& /*cache*/,
+                    const ArrayIndex& /*array_index*/,
+                    const ActionList /*meta*/,
+                    const ParallelComponent* const /*meta*/,
+                    const int value) noexcept {
+    db::mutate<ValueTag>(
+        make_not_null(&box), [&value](
+                                 const gsl::not_null<int*> value_box) noexcept {
+          *value_box = value;
+        });
+  }
+};
+
+struct simple_action_b {
+  template <typename... InboxTags, typename Metavariables, typename ActionList,
+            typename ParallelComponent, typename ArrayIndex>
+  static void apply(const db::DataBox<tmpl::list<ValueTag>>& /*box*/,
+                    const tuples::TaggedTuple<InboxTags...>& /*inboxes*/,
+                    Parallel::ConstGlobalCache<Metavariables>& cache,
+                    const ArrayIndex& /*array_index*/,
+                    const ActionList /*meta*/,
+                    const ParallelComponent* const /*meta*/,
+                    const int to_call) noexcept {
+    if (to_call == 0) {
+      Parallel::simple_action<simple_action_a>(
+          Parallel::get_parallel_component<simple_action_mock<Metavariables>>(
+              cache),
+          11);
+    } else if (to_call == 1) {
+      Parallel::simple_action<simple_action_a>(
+          Parallel::get_parallel_component<simple_action_mock<Metavariables>>(
+              cache),
+          14);
+    } else if (to_call == 2) {
+      Parallel::simple_action<simple_action_c>(
+          Parallel::get_parallel_component<simple_action_mock<Metavariables>>(
+              cache));
+    } else if (to_call == 3) {
+      Parallel::simple_action<simple_action_c>(
+          Parallel::get_parallel_component<simple_action_mock<Metavariables>>(
+              cache)[0]);
+    } else {
+      ERROR("to_call should be 0 or 1");
+    }
+  }
+};
+
+struct simple_action_c_mock {
+  template <typename... InboxTags, typename Metavariables, typename ActionList,
+            typename ParallelComponent, typename ArrayIndex>
+  static void apply(db::DataBox<tmpl::list<ValueTag>>& box,  // NOLINT
+                    const tuples::TaggedTuple<InboxTags...>& /*inboxes*/,
+                    const Parallel::ConstGlobalCache<Metavariables>& /*cache*/,
+                    const ArrayIndex& /*array_index*/,
+                    const ActionList /*meta*/,
+                    const ParallelComponent* const /*meta*/) noexcept {
+    db::mutate<ValueTag>(
+        make_not_null(&box), [](const gsl::not_null<int*> value_box) noexcept {
+          *value_box = 25;
+        });
+  }
+};
+
+struct SimpleActionMockMetavariables {
+  using component_list =
+      tmpl::list<simple_action_mock<SimpleActionMockMetavariables>>;
+  using const_global_cache_tag_list = tmpl::list<>;
+
+  enum class Phase { Initialize, Exit };
+};
+
+SPECTRE_TEST_CASE("Unit.ActionTesting.MockSimpleAction", "[Unit]") {
```

Comment:
I found this test to be confusing, possibly because I expected it to be an example of how to use a mock Action when testing another Action, and it primarily tests the consequences of calling the mock Actions (which indirectly tests that the primary Action works).  So maybe add a comment that this is more of a test that the mock Action is called rather than an example of using a mock Action (which in my mind is just an empty placeholder), or am I missing the point of the test?

Reply:
or maybe add an issue (or comment to #943) to make a better example

Reply:
The mock actions shouldn't be empty because:
1. They should provide some way of verifying they were actually called
2. They should provide some way of verifying they were called with the correct data.